ilm_ros_pkg/Color[] colors
bool[] valid
